src/compat/init_path_ge_409.ml

```diff
@@ -1 +1,4 @@
 let init_path () = Compmisc.init_path ()
+let lookup_module ~loc:_ lid env = Env.lookup_module ~load:true lid env
+let find_type env ~loc head_id = Typetexp.find_type env loc head_id
+let find_modtype env ~loc head_id = Typetexp.find_modtype env loc head_id
```

Comment:
The current design is that we have several different backward-compatibility files, one per group of compatibility: `init_path` for `Compmisc.init_path`, and `types` for (module) type representation issues. Rather than adding those lines in `init_path`, could you create a new sort of compatibility files `env_lookup_*`?